Find Food Trucks SF/Find Food Trucks SF/MapViewController.swift

```diff
@@ -83,12 +86,31 @@ extension MapViewController: MKMapViewDelegate {
         return view
     }
     
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
```

Comment:
great way of doing a segue similar to the one I made previously. 
The food truck data directly sends data from the map annotations to foodtruckdetailviewcontroller.